examples/src/java/com/twitter/heron/examples/api/ComponentJVMOptionsTopology.java

```diff
@@ -44,7 +44,7 @@ public static void main(String[] args) throws Exception {
 
     builder.setSpout("word", new TestWordSpout(), 2);
     builder.setBolt("exclaim1", new ExclamationBolt(), 2)
-        .shuffleGrouping("word");
+        .shuffleGrouping("word").addConfiguration("heron-test-config", "heron-test-key");
```

Comment:
better add a comment saying this config key-value pair for bolt level config setting demo

Reply:
Added.

Reply:
Added.